README.md

```diff
@@ -45,71 +45,71 @@ There are several **enhancer** packages, which are intended to be used in conjun
 </details>
 
 <details>
-  <summary>Web project with Enzyme</summary>
+  <summary>Web project with RTL</summary>
 
   ```js
   const { compose, baseConfig } = require('@moxy/jest-config-base');
```

Comment:
Missing 'use strict';

I think we are missing in the README files of the packages as well.

Reply:
Yep. Fixed! ✅